test/e2e/helpers.js

```diff
@@ -119,6 +119,9 @@ async function withFixtures(options, testSuite) {
       contractRegistry = ganacheSeeder.getContractRegistry();
     }
 
+    await fixtureServer.start();
+    await fixtureServer.loadJsonState(fixtures, contractRegistry);
+
```

Comment:
placing the calls here is also needed in order to mitigate the balance loading issue. It seems we need the state also earlier